### DISCUSSION TITLE
QueryInterface and automatically update "updated_at" field

### DISCUSSION THREAD

**Discussion by dana-blanc-healthy** [sequelize/sequelize#15395] [Help & Questions] [answered]
Hello!

Is there a way to automatically update the updated_at timestamp that Sequelize manages when using QueryIntreface?
All the solutions I found only manage the updated_at when working with models.

For example, this query doesn't update the `updated_at` field.
```
  up: (queryInterface) =>
    queryInterface.sequelize.query(`update "Products" set price = 5 where price = 6`, {
      type: queryInterface.sequelize.QueryTypes.UPDATE,
    }),
```

Thank you!

**Comment by ephys** [comment, marked as answer]
Hello :)

Unfortunately not automatically. They're updated by Sequelize and are part of the query.
You'd need to include `updated_at` in your query

It would be interesting to add the possibility of having `updated_at` and `created_at` be handled by the database using triggers imo
